Formula/clasp-cl.rb

```diff
@@ -0,0 +1,34 @@
+class ClaspCl < Formula
+  desc "Common Lisp implementation that brings Common Lisp and C++ together"
+  homepage "https://github.com/clasp-developers/clasp"
+  url "https://github.com/clasp-developers/clasp/archive/refs/tags/2.0.0.tar.gz"
+  sha256 "de3c5efaad0ca1e87ce58e937a45f2740d0c5cd869a500b606eb9e2ea37cb837"
+  license "GPL-2.0-or-later"
+  head "https://github.com/clasp-developers/clasp.git",
+       using:  :git,
+       branch: "main"
+
+  depends_on "boost" => :build
+  depends_on "libunwind-headers" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "sbcl" => :build
+  depends_on arch: :x86_64
+  depends_on "fmt"
+  depends_on "gmp"
+  depends_on "llvm@14"
+
+  def install
+    ENV.deparallelize
+    system "./koga", "--reproducible-build", "--bin-path=#{bin}", "--share-path=#{share}/clasp/",
+      "--lib-path=#{lib}/clasp/", "--llvm-config=#{Formula["llvm@14"].opt_bin}/llvm-config",
+      "--pkg-config=#{Formula["pkg-config"].opt_bin}/pkg-config",
+      "--cxxflags=-I#{Formula["boost"].include}/", "--cppflags=-I#{Formula["boost"].include}/",
+      "--skip-sync=ansi-test,mps,cl-bench,cl-who"
```

Comment:
Can't it find boost with pkg-config?
```suggestion
    system "./koga", "--reproducible-build",
                                "--bin-path=#{bin}",
                                "--share-path=#{pkg_share}",
                                "--lib-path=#{lib}/clasp/",
                                "--llvm-config=#{Formula["llvm@14"].opt_bin}/llvm-config",
                                "--pkg-config=#{Formula["pkg-config"].opt_bin}/pkg-config",
                                "--cxxflags=-I#{Formula["boost"].include}/",
                                "--cppflags=-I#{Formula["boost"].include}/",
                                "--skip-sync=ansi-test,mps,cl-bench,cl-who"
```

Reply:
patching in C flags to find headers doesn't sound like a good option. Is there no better way to specify this?

Reply:
koga is our metabuilder. It is basically our version of "configure." It determines what the cflags for the build will be so we aren't really patching the flags.

I can add an option to specifically set the boost home, but I'd have to do release like 2.1.0 or 2.0.1. Is there anyway we can leave that for now?

Reply:
> It determines what the cflags for the build will be so we aren't really patching the flags.

Then I'm not sure what the `cxxflags` and `cppflags` option does?

> I can add an option to specifically set the boost home

It would be much preferable to leave that to your "configure" than to a Homebrew specific cflag.

Reply:
Yes, we don't appear to need that on Mac, only Homebrew's Linux build for some reason.

Reply:
Boost doesn't ship any `.pc` files, so it cannot be found by `pkg-config`. Setting `cxxflags` and `cppflags` seems reasonable -- that's what they're there for, after all (e.g. instructing the compiler to check custom install paths not checked by default).